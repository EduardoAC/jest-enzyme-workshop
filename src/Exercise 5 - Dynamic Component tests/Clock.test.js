import React from 'react'
import { shallow } from 'enzyme'
import { Clock } from './Clock'


describe('<clock />', () => {
  /**
   * 
   * For this exercise you will have to use jest-date-mock
   * It's already in the project so follow the usage instructions
   * https://github.com/hustcc/jest-date-mock#readme
   * 
   */
  it('should display the right time', () => {
    const wrapper = shallow(<Clock />)
    expect(wrapper).toMatchSnapshot()
  })
})