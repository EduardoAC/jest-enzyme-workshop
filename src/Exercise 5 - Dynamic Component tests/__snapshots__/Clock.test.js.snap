// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<clock /> should display the right time 1`] = `
<p>
  18:33:52
</p>
`;
